doc/nrf/ug_thread_certification.rst

```diff
@@ -58,21 +58,15 @@ Complete the following steps to run the certification tests:
 
 #. Build the certification image.
 
-   The :ref:`ot_cli_sample` sample is used as a base, modified with an overlay file.
-
+   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` overlay file.
```

Comment:
You need a blank line after this and before the code-block, or it will not show up correctly.